src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedAtlas.java

```diff
@@ -227,6 +239,8 @@
     private final LongToLongMultiMap relationIndexToRelationIndices;
     private final LongToLongMultiMap relationOsmIdentifierToRelationIdentifiers;
     private final LongArray relationOsmIdentifiers;
+    private ByteArrayOfArrays relationGeometries;
```

Comment:
I would call this one `relationIndexToRelationGeometry`

Reply:
If that is what it is actually...

Reply:
So to make this consistent, I would make this some `MultipolygonArray` (which would need to be created) on the model of `PolygonArray` for areas.
Then I would get rid of `builtRelationGeometries` below, and use the array directly in accessor methods like
```
protected MultiPolygon relationGeometry(final long index)
```
below.

Reply:
Hmm, I actually think this is a bad idea. Currently, we're getting a lot of mileage out of the fact that a WKT for a multipolygon is pretty small once compressed, and it has all the associated mappings of outers to inners already built in. We can't use the PrimitivePointsArray like PolygonArray does easily, because to do that we'd have to do the work to serialize holes ourselves, which would mean an array of outers which are themselves arrays as well, so it would be an array of arrays of arrays..... when in this case a single 2d array works quite well. I'm willing to hear the argument that it's necessary, but performance and size are quite capable as is so I'm not convinced upfront.

Reply:
I am inclined to agree with Sam - from a serialization perspective it's much clearer and simpler to just store the WKT. The array loads really quickly and it also means we can build the polygon on demand. Otherwise we'd have to spend a bunch of time designing some kind of proto object and dealing with all that nonsense.